package.json

```diff
@@ -42,6 +42,7 @@
     "debug": "^4.0.0",
     "find-root": "1.1.0",
     "fs-extra": "^3.0.1",
+    "gelf-stream": "^1.1.1",
```

Comment:
Just fyi, about dependency versions. We usually go with exact versioning in Ghost & Ghost-Admin and I think the same will be done in our other packages. Especially now that we have renovate bot which bumps versions as needed. Think this can be left here and this whole file needs to be changed at once to exact version :)

Reply:
Indeed

Reply:
We usually use carets (^) in sub packages of Ghost to achieve installing one version for multiple packages. Sub packages are Ignition, knex-migrator etc..

If we pin a version to an exact version here, we would end in a situation where Ghost needs to e.g. install 10x lodash because each sub dependency could use a different lodash pinned version.

We use exact versions in Ghost.

Reply:
Oh. Thanks for explaining @kirrg001 So, we only optimize Ghost/Ghost-Admin with strict dependencies the rest is `^` to save time/space during package installation in Ghost :+1: